portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,17 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+body {
+    background-color: #080808;
+    background: -webkit-canvas("mycanvas");
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 98%;
 }
 
-#greeting-container {
-  margin-top: 20px;
+#mycanvas { 
+  display: block; width: 100vw; height: 100vh;
+}
+
+button {
+    position: absolute;
+    z-index: 1;
```

Comment:
you can probably (hopefully) remove this now. since the canvas is now behind everything.

Generally you want to mess with z-index as little as possible.